osbenchmark/workload/params.py

```diff
@@ -63,6 +65,14 @@ def param_source_for_name(name, workload, params):
     else:
         return param_source(workload, params)
 
+def get_standard_value_source(op_name, field_name):
+    try:
+        return __STANDARD_VALUE_SOURCES[op_name][field_name]
+    except KeyError:
+        raise exceptions.SystemSetupError(
+            "Could not find standard value source for operation {}, field {}! Make sure this is registered in workload.py"
```

Comment:
Nice! This is a helpful suggestion. It'd be good for us to add documentation related to this in the official docs. Once we have that, we can add a link to that guide to this error message.